W-15612083-Run-VALE-on-latest-Studio-repo-CP (#524)
mulesoft/docs-studio: modules/ROOT/pages/breakpoints-concepts.adoc

. Right-click the message processor and select *Add Breakpoint*:
+
image::add-breakpoint.png[]
. In the *Mule Breakpoints* view, select the breakpoint to which you want to add a condition, and select *Conditional*:
+
image::set-conditional-breakpoint.png[]
. Enter the condition that must be met to activate the breakpoint. +
For example, the following expression activates the breakpoint if the method used in the request is `GET`:
+
[source]
----
attributes.method == "GET"
----

> You must not stop the application's execution to change the condition.

Reviewer: ```suggestion
You can change the condition without stopping the application's execution.
```
I think it's more clear this way.

Author: OK! Thanks!
